src/views/ProductsDetails.vue

```diff
@@ -198,25 +201,57 @@ const submitProduct = async () => {
     productsContent: productsContent.value,
   };
   try {
-    await store.dispatch("products/createProduct", product);
-    return true;
+    const res = await store.dispatch("products/createProduct", product);
+    snackbarProvider.showSuccessSnackbar("Successfully added product!");
+    return res;
   } catch (error) {
     snackbarProvider.showErrorSnackbar("Could not create the product");
   }
   return false;
 };
 
+const handlePictureSelected = (chosenFile) => {
+  selectedPicture.value = chosenFile;
+};
+
+const isPictureValidated = () => {
+  if (selectedPicture.value <= 0) {
+    return false;
+  }
+  return true;
+};
+
+const postPicture = async (id, image) => {
+  try {
+    await store.dispatch("products/postPicture", { productId: id, image });
+    snackbarProvider.showSuccessSnackbar(
+      "Successfully added product and picture!"
+    );
+  } catch (error) {
+    snackbarProvider.showErrorSnackbar(
+      "Couldn't add the picture to the product!"
+    );
+  }
+};
+
+async function submitPicture(productResponse, selectedImage) {
```

Comment:
You can remove `selectedImage` and use `selectedPicture.value` instead.

Reply:
done